ui/components/app/connected-accounts-permissions/connected-accounts-permissions.js

```diff
@@ -55,7 +55,7 @@ const ConnectedAccountsPermissions = ({ permissions }) => {
                 className="connected-accounts-permissions__checkbox"
               />
               <label htmlFor={permissionName}>
-                {getPermissionDescription(t, permissionName).label}
+                {getPermissionDescription(t, permissionName)[0].label}
```

Comment:
Is this guaranteed to be a single item?

Reply:
This only applies to DApps right now which only have single item permissions, so for now yeah.